cpp/pybind/CMakeLists.txt

```diff
@@ -102,6 +102,13 @@ set_target_properties(${PACKAGE_NAME} PROPERTIES
                       FOLDER "Python"
                       LIBRARY_OUTPUT_DIRECTORY "${PYTHON_COMPILED_MODULE_DIR}"
                       ARCHIVE_OUTPUT_DIRECTORY "${PYTHON_COMPILED_MODULE_DIR}")
+# Include additional libraries that may be absent from the user system
+# eg: libc++.so and libc++abi.so (needed by filament) for Linux.
+# libc++.so is a linker script including libc++.so.1 and libc++abi.so, so append 1 to libc++.so
```

Comment:
In my case, they are symbolic links:

```
lrwxrwxrwx 1 root root   14 Nov 22  2018 libc++abi.so -> libc++abi.so.1
lrwxrwxrwx 1 root root   16 Nov 22  2018 libc++abi.so.1 -> libc++abi.so.1.0
-rw-r--r-- 1 root root 203K Nov 22  2018 libc++abi.so.1.0

-rw-r--r-- 1 root root   28 Nov 22  2018 libc++.so
lrwxrwxrwx 1 root root   13 Nov 22  2018 libc++.so.1 -> libc++.so.1.0
-rw-r--r-- 1 root root 757K Nov 22  2018 libc++.so.1.0
```

So in this system, we need ` libc++abi.so.1.0` and `libc++.so.1.0`. Would it be possible for CMake to resolve the symbolic links, instead of manually append the `.` at the end?

Reply:
`libc++.so` is not a symlink, unlike `libc++abi.so`, so it will not resolve to `libc++.so.1`. It's a script that needs to be parsed by the linker to find the actual library. `cmake` does not have functionality to do this.
The current method fixes this in a hacky way, but it will work as long as `libc++.so` has the same major ABI version. This is not needed for `libc++abi.so`, which is a symlink.